ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -36,6 +48,11 @@ function ViolationsTablePage(): ReactElement {
     const [searchOptions, setSearchOptions] = useState<string[]>([]);
     const { searchFilter, setSearchFilter } = useURLSearch();
 
+    const [activeViolationStateTab, setActiveViolationStateTab] = useURLStringUnion(
```

Comment:
We can pass this as a context for autocomplete options. I found the results weird, so I decided not to do it yet. For some reason, it kept showing only 1 autocomplete option when there should be more. I'll have to look into that.

Reply:
Weird - I definitely think this would be useful to have. Maybe this specific field isn't well supported by the API currently?

Reply:
Maybe. I found it weird that the autocomplete options aren't always returned with a fixed amount of options either. I've tried passing in different values for `query` and `category`. Sometimes it'll return 7, 8, 9, or even 1 result 🤔